item-impl/src/test/java/com/example/auction/item/impl/ItemServiceImplIntegrationTest.java

```diff
@@ -41,7 +41,7 @@
  */
 public class ItemServiceImplIntegrationTest {
 
-    private final static Setup setup = defaultSetup().withCassandra()
+    private final static Setup setup = defaultSetup().withCassandra(true)
```

Comment:
Why are we putting back the `withCassandra(true)`. In Lagom 1.4.0 we make this method less verbose by allowing `withCassandra()`. 

It doesn't make much sense to have `withCassandra(false)` because is the default, so `with` and `true` is redundant.

Reply:
You're right. I was getting compilation errors in IntelliJ and just went ahead. I had not reloaded the project in a while so it was was false negatives.